angr/simos/simos.py

```diff
@@ -2,7 +2,7 @@
 
 import claripy
 from archinfo import ArchMIPS32
-
+import struct
```

Comment:
This line should go right below `import logging`.

At the beginning of each Python file, we usually following the following import style:

```
built-in packages
- a new line
third-party non-angr packages
- a new line
angr-packages
- a new line
all imports from the current package
```